make/autoconf/jdk-options.m4

```diff
@@ -24,7 +24,7 @@
 #
 
 # ===========================================================================
-# (c) Copyright IBM Corp. 2020, 2020 All Rights Reserved
+# (c) Copyright IBM Corp. 2020, 2021 All Rights Reserved
```

Comment:
Is there an IBM specific change to this file in this PR, why change the Copyright?

Reply:
Yes, there still are changes relative to openjdk.

Reply:
The only change made to this `jdk-options.m4` file was:
from
```
    # Use the date from version-numbers
```
to
```
    # Use the date from version-numbers.conf
```
This is not IBM specific, do I miss something?

Reply:
It won't stop me merging, but it does seem wrong that the only change to this file in the "Replace hooks in CompileJavaModules.gmk" commit is the copyright update. The IBM copyright shouldn't need to be updated due to OpenJDK changes. The IBM copyright won't get updated for everything that merges cleanly.

Reply:
That file was changed in the merge commit, but I forgot to update the copyright date at that time.